data/templates/new_game_template

```diff
@@ -6,11 +6,15 @@
 
 # Section `global`: General game settings.
 [global]
+# Whether a multiplayer game should be configured. Default: false
+# Note, that this cannot be used for a real multiplayer game, because the host will be the only possible human player
```

Comment:
Sounds a bit unclear IMHO. Suggestion:
```suggestion
# Note that this cannot be used for a real multiplayer game, because the game will not be registered on
# the metaserver or in the local network, and therefore no clients or spectators will be able to join.
```